PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/spark/common/SparkSubmitModel.java

```diff
@@ -269,10 +277,6 @@ public void finished(boolean aborted, int errors, int warnings, CompileContext c
     }
 
     private void showCompilerErrorMessage(@NotNull CompileContext compileContext) {
-        postEventProperty.put("IsSubmitSucceed", "false");
```

Comment:
Why remove these? The method `buildArtifactObservable` still calls the function, which would lost the telemetry point.